src/components/DistanceRequest/index.js

```diff
@@ -229,7 +229,4 @@ export default withOnyx({
     transaction: {
         key: ({transactionID}) => `${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID || 0}`,
     },
-    mapboxAccessToken: {
-        key: ONYXKEYS.MAPBOX_ACCESS_TOKEN,
-    },
```

Comment:
This is a cleanup change, discussed [here](https://github.com/Expensify/App/pull/26307#discussion_r1347867780).

Reply:
Thank you!